src/api/routes.ts

```diff
@@ -277,6 +277,13 @@ export const buildPostInvitationsRoute = (itemId: UUID) =>
 export const buildResendInvitationRoute = (args: { itemId: UUID; id: UUID }) =>
   `${ITEMS_ROUTE}/${args.itemId}/${INVITATIONS_ROUTE}/${args.id}/send`;
 
+export const buildItemPublishRoute = (itemId: UUID, notification: boolean) => {
+  const queryString = notification
+    ? qs.stringify({ notification }, { addQueryPrefix: true })
+    : '';
```

Comment:
If notification is not defined, `qs` will result to `''` automatically, so you don't need to check the value of `notification`. If it doesn't work with your implementation, probably you need to force in the backend at the endpoint level that `notification` is a boolean (maybe with the schema).